python/pyarrow/pandas_compat.py

```diff
@@ -172,7 +172,10 @@ def get_column_metadata(column, name, arrow_type, field_name):
 
     return {
         'name': name,
-        'field_name': str(field_name),
+        'field_name': (
+            'None' if field_name is None
+            else _column_name_to_strings(field_name)
```

Comment:
Is this call to `_column_name_to_strings` necessary? I would think that was already done for field_name when you end up here

Reply:
Nope, removed it.